### ISSUE TITLE
Server-side rendering cannot find componentHandler

### ISSUE THREAD

**Issue by AugustinLF** [tleunen/react-mdl#193]
I have a test set up using JSDOM and react-addons-test-utils (similar to the one used by [react-redux](https://github.com/rackt/react-redux/blob/master/test/setup.js)). However, the tests break because I cannot render any react-mdl component. Do you know how to make it work? I've seen #158, but I don't think that can solve my problem!

**Comment by tleunen** [comment]
Do you include the material.js file before running your tests?

**Comment by AugustinLF** [comment]
Yes I did a `require('react-mdl/extra/material');`. Then, I get a `Element is not defined` at line 187 of material.js `if (!(typeof element === 'object' && element instanceof Element)) {`.

I get that this does not seem to be a react-mdl problem, but more a problem to render any mdl Element in a JSDOM context, where Element is not a global.

I solved that by adding a `global.Element = global.window.Element;` after my JSDOM setup. Sorry for bothering you!
